### PR TITLE
Sagitta Some optimization adjustments

### PR DESCRIPTION
**Pull request by sofukong**
Some optimization adjustments

### CODE REVIEW

File: docs/contributing/build-vyos.rst [hunk 1]
```diff
@@ -23,6 +23,50 @@ also set up your own build machine and run a :ref:`build_native`.
...
+
+Native Build
+============
+
+To build VyOS natively you require a properly configured build host with the
+following Debian versions installed:
+
+- Debian Jessie for VyOS 1.2 (crux)
+- Debian Buster for VyOS 1.3 (equuleus) 
+- Debian Bullseye for VyOS 1.4 (sagitta)
```
Comment (dmbaturin) [inline docs/contributing/build-vyos.rst]: It's 12/Bookworm for Sagitta now, not Bullseye.

File: docs/contributing/build-vyos.rst [hunk 2]
```diff
@@ -23,6 +23,50 @@ also set up your own build machine and run a :ref:`build_native`.
...
+Native Build
+============
+
+To build VyOS natively you require a properly configured build host with the
+following Debian versions installed:
+
+- Debian Jessie for VyOS 1.2 (crux)
+- Debian Buster for VyOS 1.3 (equuleus) 
+- Debian Bullseye for VyOS 1.4 (sagitta) 
+- Debian unknown for VyOS 1.5 (circinus, current) - aka the rolling release
```
Comment (dmbaturin) [inline docs/contributing/build-vyos.rst]: It's also Bookworm for current. I think a better wording might be "Debian Bookworm for the upcoming 1.5/Circinus (subject to change)".

**Review by dmbaturin — CHANGES_REQUESTED** [review]

**Review by dmbaturin — APPROVED** [review]
Sorry, I overlooked the change in a later commit. It's a good idea to squash or force push such changes to make them easier to review.

File: docs/contributing/build-vyos.rst (hunk 1 above)
Comment (sofukong) [reply docs/contributing/build-vyos.rst]: 我对GitHub还不太熟悉，我会在后面改过来，谢谢！

Comment (sofukong) [reply docs/contributing/build-vyos.rst]: I am not very familiar with GitHub yet, I will make changes later. Thank you!

File: docs/contributing/build-vyos.rst (hunk 2 above)
Comment (sofukong) [reply docs/contributing/build-vyos.rst]: I am not very familiar with GitHub yet, I will make changes later. Thank you!

**Comment by c-po** [comment]
Fixed in https://github.com/vyos/vyos-documentation/pull/1213

**Pull request closed without merging**